### ISSUE TITLE
Insufficient contrast ratio in notifications

### ISSUE THREAD

**Issue by Comandeer** [ckeditor/ckeditor4#5495]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Open https://codepen.io/Comandeer/pen/OJabbmX
2. Click the "Show notification" button.

### Expected result

A link inside the notification should have enough contrast ratio – preferably it should have the same color as the rest of the text (it's sufficiently differentiated from it thanks to the underline).

### Actual result

The link has [too low contrast ratio](https://www.siegemedia.com/contrast-ratio#%2312306f-on-%23c83939).

## Other details

* Browser: N/A
* OS: N/A
* CKEditor version: N/A
* Installed CKEditor plugins: notification
